package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "bedrock-web-payment",
+  "version": "0.0.1",
+  "description": "Client side utilities for bedrock-payment.",
+  "main": "lib/index.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/digitalbazaar/bedrock-web-payment.git"
+  },
+  "keywords": [
+    "payment",
+    "bedrock",
+    "web",
+    "client",
+    "javascript"
+  ],
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "https://digitalbazaar.com"
+  },
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/bedrock-web-payment/issues"
+  },
+  "homepage": "https://github.com/digitalbazaar/bedrock-web-payment#readme",
+  "peerDependencies": {
+    "bedrock-payment": "^0.0.1",
```

Comment:
Historically, front end modules do not specify back-end dependencies or peer deps, for example: https://github.com/digitalbazaar/bedrock-web-account/blob/master/package.json

Reply:
It might be a good idea to question this convention as I think the warning about not having the peerDependency `bedrock-payment-http` might be a good idea as it would inform a developer that you installed a front end module, but forgot the back end module. However this is `bedrock-payment` which is no longer needed.

Reply:
There is no requirement that the front-end and back-end live on the same server.  In fact we look forward to the day when the front-end bundle can live on a CDN.

Reply:
That is a valid point. Removed all peerDependencies here: https://github.com/digitalbazaar/bedrock-web-payment/pull/1/commits/11f3885c9cfb225ebbecb915db0a33afd46a99ff